exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/ElasticsearchExporterFaultToleranceIT.java

```diff
@@ -11,25 +11,48 @@
 
 import io.camunda.zeebe.protocol.record.Record;
 import io.camunda.zeebe.test.util.record.RecordingExporter;
-import io.camunda.zeebe.test.util.socket.SocketUtil;
 import java.time.Duration;
 import java.util.stream.Collectors;
 import org.awaitility.Awaitility;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
+import org.testcontainers.containers.ToxiproxyContainer;
+import org.testcontainers.containers.ToxiproxyContainer.ContainerProxy;
+import org.testcontainers.utility.DockerImageName;
 
 public class ElasticsearchExporterFaultToleranceIT
     extends AbstractElasticsearchExporterIntegrationTestCase {
 
+  // we use Toxiproxy to simulate Elastic being "down" arbitrarily
```

Comment:
Well, the proxy address exists, but there is no upstream yet (upstream being Elastic). So it's effectively down, as in, all requests are rejected.

This is actually temporary, since once https://github.com/camunda/zeebe/pull/9380 is merged, we don't use Toxiproxy but just a plain socat container. But the general idea is the same, you start a proxy but not the upstream, and only start upstream later.